src/osyris/spatial/__init__.py

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Osyris contributors (https://github.com/nvaytet/osyris)
+
+# flake8: noqa
+
+from .spatial import *
```

Comment:
Yeah that can be done. Since most of the spherical/cylindrical component computations have been inlined in vector.py, the spatial module will only have `change_origin` and `change_basis` in it.